.eslintrc.cjs

```diff
@@ -1,16 +1,21 @@
 module.exports = {
+  root: true,
   env: {
     es2021: true,
     node: true,
   },
-  extends: ['eslint:recommended', 'plugin:n/recommended'],
+  extends: ['eslint:recommended', 'plugin:n/recommended', 'plugin:@typescript-eslint/recommended'],
+  parser: '@typescript-eslint/parser',
```

Comment:
We'll be running all ts and js (so not only ts) files through the parser now. It should help us to surface more possible bugs and issues because eslint typescript plugin includes many useful rules. In my opinion that's what we should do, but I'm open to the feedback!